src/web/frontend/components/Chat/chatInput.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import './chat.scss';
+import KeyboardArrowRightIcon from '@material-ui/icons/KeyboardArrowRight';
+
+interface InputProps {
+    onSubmit: any,
```

Comment:
don't use any, if it's a function - use type alias
https://www.typescriptlang.org/docs/handbook/advanced-types.html#type-aliases